### ISSUE TITLE
CSS: button backdrop filter should be positive brightness on dark mode for accessibility (like cobalt.tools)

### ISSUE THREAD

**Issue by wheatwhole** [cyclonelib/cyclonelib#20]
add a proper dark variant for the button backdrop filter
